animal.go

```diff
@@ -32,9 +32,11 @@ func GetUserYesOrNo(question string) string {
 	scanner := bufio.NewScanner(os.Stdin)
 	scanner.Scan()
 	input := scanner.Text()
-	fmt.Printf("You typed: %q", input)
+	fmt.Printf("You typed: %s\n", input)
 	if input == "yes" {
+		//fmt.Printf("Returning: %s", input)
 		return "yes"
```

Comment:
It's usually a good idea to use named constants instead of literal values for things like this. Then if you ever decide to change what the string actually is, you don't need to change the code everywhere: you just change the constant definition. For example:

```go
const (
	AnswerYes = "yes"
	AnswerNo = "no"
)
```

Reply:
Added, thank you for the suggestion.